server.go

```diff
@@ -194,6 +197,17 @@ func (dd *Devd) WrapHandler(log termlog.TermLog, next httpctx.Handler) http.Hand
 				}
 			}
 		}
+		if dd.Cors {
```

Comment:
Don't follow when this is set to true

Reply:
I don't understand your remark here.

Reply:
Sorry i misread `Cors: *cors` in `devd.go` as if it was struct for some reason... it's a bool ref i see now